src/kinetics/InterfaceKinetics.cpp

```diff
@@ -595,4 +595,30 @@ void InterfaceKinetics::setPhaseStability(const size_t iphase, const int isStabl
     }
 }
 
+double InterfaceKinetics::InterfaceCurrent(const size_t iphase)
+{
+    int nSp = thermo(iphase).nSpecies();
+    double charge_k[nSp];
+    double sdot_k[nSp];
```

Comment:
These should be `vector_fp`'s, as sizes aren't known at compile time.

Reply:
Changed both of these to `vector_fp` type.